### ISSUE TITLE
Panes steal focus when resizing

### ISSUE THREAD

**Issue by Aerijo** [atom/atom#20216]
### Prerequisites

* [X] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: <https://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode>
    * Followed all applicable steps in the debugging guide: <https://flight-manual.atom.io/hacking-atom/sections/debugging/>
    * Checked the FAQs on the message board for common solutions: <https://discuss.atom.io/c/faq>
    * Checked that your issue isn't already filed: <https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom>
    * Checked that there is not already an Atom package that provides the described functionality: <https://atom.io/packages>

### Description

When the pane contents is something like an iframe, it will steal focus and stop the drag when resizing panes. This is relevant for embedded PDF viewing (especially relevant to LaTeX use), as the viewer will steal focus and prevent the PDF view from being resized properly.

### Steps to Reproduce

1. Have the following in your `init.js` file
```javascript
atom.workspace.addOpener(uri => {
  if (uri === "frame://example") {
    const frame = document.createElement("iframe");
    frame.setAttribute("style", "width:100%;height:100%");

    return {
      getTitle: () => "iframe",
      element: frame,
    };
  }
})

atom.workspace.addOpener(uri => {
  if (uri === "editor://example") {
    return atom.workspace.buildTextEditor();
  }
})

atom.workspace.open("frame://example");
atom.workspace.open("editor://example");
```
2. Move the TextEditor panel so that both are visible side by side
3. Resize the panels by dragging the resize element towards the iframe.

**Expected behavior:**

The iframe panel gets smaller

**Actual behavior:**

The panel does not get smaller unless you drag it really slowly

**Reproduces how often:**

100%

### Versions

Atom    : 1.41.0
Electron: 4.2.7
Chrome  : 69.0.3497.128
Node    : 10.11.0

Ubuntu 19.04

### Additional Information

[PR here](https://github.com/atom/atom/pull/19862)

**Comment by allefeld** [comment]
@Aerijo, we really seem to be synchronized right now. :) I came across the same problem.

I thought about whether it would be possible to forward `mousemove` events from inside the `iframe` to the main DOM tree, but your approach in the PR of adding a `div` across the whole window is much more elegant. Thanks for working on this, and let's hope your PR gets accepted.

**Comment by Aerijo** [comment]
This was fixed by #19862
